src/panels/lovelace/editor/config-elements/hui-weather-forecast-card-editor.ts

```diff
@@ -86,7 +141,54 @@ export class HuiWeatherForecastCardEditor
             { name: "theme", selector: { theme: {} } },
           ],
         },
-        ...(hasForecast
+        ...(!hasForecastLegacy &&
```

Comment:
I don't like all those `...`. But I guess it's existing code...

Reply:
Yeah, let's not mess with that in this PR